spec/Section 6 -- Execution.md

```diff
@@ -648,7 +648,7 @@ CompleteValue(fieldType, fields, result, variableValues):
     * If {completedResult} is {null}, throw a field error.
     * Return {completedResult}.
   * If {result} is {null} (or another internal value similar to {null} such as
-    {undefined} or {NaN}), return {null}.
+    {undefined}), return {null}.
```

Comment:
This change is technically a normative change, not just editorial. However I did some checking and found the reference implementation (and most others) do not adhere to short-circuiting NaN to null.

Reply:
GraphQL.NET throws on Nan.